docs/overview/office-add-ins-fundamentals.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Building Office Add-ins
-description: 'An introduction to developing Office Add-ins.'
+description: 'An introduction to building Office Add-ins.'
```

Comment:
I'd prefer "developing", because "building" is a loaded term in programming that means something akin to compiling/preparing, not developing.

Reply:
@Rick-Kirkham The build is complaining because this file and the [Developing Office Add-ins](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-build-suggestions/docs/develop/develop-overview.md) file have the same metadata description. I'm open to suggestions for either/both.

Reply:
OK. I see that we use "build" to mean "develop" all over our docs, so forget my comment.